aruco_detect:
  ros__parameters:
    adaptiveThreshConstant: 7.0
    adaptiveThreshWinSizeMin: 3
    adaptiveThreshWinSizeMax: 53
    adaptiveThreshWinSizeStep: 4
    cornerRefinementMaxIterations: 30
    cornerRefinementMinAccuracy: 0.01
    cornerRefinementWinSize: 5
    doCornerRefinement: true
    cornerRefinementSubpix: true
    errorCorrectionRate: 0.6
    minCornerDistanceRate: 0.05
    markerBorderBits: 1
    maxErroneousBitsInBorderRate: 0.04
    minDistanceToBorder: 3
    minMarkerDistanceRate: 0.1
    minMarkerPerimeterRate: 0.03
    maxMarkerPerimeterRate: 4.0
    minOtsuStdDev: 5.0
    perspectiveRemoveIgnoredMarginPerCell: 0.13
    perspectiveRemovePixelPerCell: 8
    polygonalApproxAccuracyRate: 0.01
